yoyodyne/data/collators.py

```diff
@@ -19,9 +19,9 @@ class Collator:
 
     has_features: bool
     has_target: bool
-    max_source_length: int = defaults.MAX_SOURCE_LENGTH
-    max_features_length: int = defaults.MAX_FEATURES_LENGTH
-    max_target_length: int = defaults.MAX_TARGET_LENGTH
+    max_source_length: int = defaults.MAX_LENGTH
+    max_features_length: int = defaults.MAX_LENGTH
+    max_target_length: int = defaults.MAX_LENGTH
```

Comment:
Feature length makes sense but target and source should be different length vars. (Zh/Ko/Ja tokens)

Reply:
They can be, they just have the same default value. (And always did, I just sloppily gave them different names a long time ago.)